### DISCUSSION TITLE
Ender 3 S1 - MAX_CURRENT ?

### DISCUSSION THREAD

**Discussion by wwian** [mriscoc/Ender3V2S1#528] [Q&A]
I have an Ender 3 S1 with CR-FDM-v24S1_301, stock Creality stepper motors. 

I want to adjust the current going to my Z stepper motor. In reviewing configuration.adv I found under _TMC26X Stepper Driver options_ a setting for MAX_CURRENT.

Are these settings for general steppers or only for the TMC26X Stepper Driver still affect the Creality stepper motors?

**Comment by mriscoc** [comment]
Creality boards doesn't support programmable current adjustment, only manual.
